### DISCUSSION TITLE
bex build broken due to 'Could not resolve "quasar/src/utils/uid"'?

### DISCUSSION THREAD

**Discussion by evandor** [quasarframework/quasar#17123] [CLI - BEX mode] [answered]
Hi everybody, 

I ran into an issue today and was able to reproduce it with a brand new quasar app (created with 'yarn create quasar' and using all the defaults (and typescript)). When you add the bex mode and run 'quasar dev -m bex', the build will fail with 

 App •  WAIT  • Compiling of Background Script with Esbuild in progress...
✘ [ERROR] Could not resolve "quasar/src/utils/uid"

    .quasar/bex/bridge.js:8:16:
      8 │ import uid from 'quasar/src/utils/uid'
        ╵                 ~~~~~~

Before I add an issue, I'd like to reach out here to understand if anyone else has the same behaviour. I'm on Mac (M1), here's the output of quasar info:

Operating System - Darwin(23.4.0) - darwin/arm64
NodeJs - 18.19.1

Global packages
  NPM - 10.2.4
  yarn - 1.22.19
  @quasar/cli - 2.4.0
  @quasar/icongenie - 4.0.0
  cordova - 12.0.0 (cordova-lib@12.0.1)

Important local packages
  quasar - 2.15.3 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app-vite - 1.8.1 -- Quasar Framework App CLI with Vite
  @quasar/extras - 1.16.11 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.4.23 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.3.2
  pinia - Not installed
  vuex - Not installed
  vite - 2.9.18 -- Native-ESM powered web dev build tool
  eslint - Not installed
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  None installed

Thanks so much!

**Comment by rstoenescu** [comment, marked as answer]
This has been reported already. And fixed. Upgrade to latest q/app-vite or q/app-webpack
